<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveImageFileClass</class>
 <widget class="QDialog" name="MoveImageFileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoveImageFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>MoveImageFile</string>
     </property>
     <widget class="QCheckBox" name="checkBox_originalFolder">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>original Folder</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sourceFolder">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sourceFolder">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Source Folder:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_execute">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_destFolder">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dest Folder:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>File Type:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_destFolder">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_fileType">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <zorder>checkBox_originalFolder</zorder>
     <zorder>lineEdit_sourceFolder</zorder>
     <zorder>pushButton_sourceFolder</zorder>
     <zorder>label</zorder>
     <zorder>pushButton_execute</zorder>
     <zorder>pushButton_destFolder</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_destFolder</zorder>
     <zorder>label_3</zorder>
     <zorder>comboBox_fileType</zorder>
     <zorder>progressBar</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
